examples/bri-3/package.json

```diff
@@ -31,11 +31,19 @@
     "@nestjs/common": "^9.0.0",
     "@nestjs/core": "^9.0.0",
     "@nestjs/cqrs": "^9.0.1",
+    "@nestjs/jwt": "^9.0.0",
+    "@nestjs/passport": "^9.0.0",
     "@nestjs/platform-express": "^9.0.0",
     "@prisma/client": "^4.4.0",
+    "bip39": "^3.0.4",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.13.2",
+    "eslint-config-react-app": "^7.0.1",
+    "ethers": "^5.7.2",
     "ew-precise-proofs-js": "^1.2.0",
+    "passport": "^0.6.0",
+    "passport-jwt": "^4.0.0",
+    "passport-local": "^1.0.0",
```

Comment:
@ognjenkurtic unless I read the docs wrong, you need local to extend the chosen strategy, namely jwt, in order to be able to do relevant configurations of the strategy through injecting a metadata object in the strategy -- which we will probably want to do soon since we need resolvable user ids. Hence, I think it is advisable to just do the extension pattern at this point. thoughts? cc @skosito @Manik-Jain @Kasshern